bedrock/firefox/urls.py

```diff
@@ -150,6 +150,27 @@
     page("firefox/welcome/17b/", "firefox/welcome/page17/page17-b.html", ftl_files=["firefox/welcome/page14"], active_locales=["en-US", "fr", "de"]),
     page("firefox/welcome/17c/", "firefox/welcome/page17/page17-c.html", ftl_files=["firefox/welcome/page14"], active_locales=["en-US", "fr", "de"]),
     page("firefox/welcome/18/", "firefox/welcome/page18.html"),
+    page(
+        "firefox/welcome/19/",
+        "firefox/welcome/page19.html",
+        active_locales=[
+            "en-US",
+            "ar",
+            "cs",
+            "de",
+            "el",
+            "es-ES",
+            "fr",
+            "hu",
+            "id",
+            "it",
+            "ja",
+            "pl",
+            "pt-BR",
+            "ru",
+            "zh-CN",
+        ],
+      ),
```

Comment:
This should now pass ruff, sry, the suggestion ui ate my whitespace so i closed it wrong:
```suggestion
    ),
```